Pipfile

```diff
@@ -56,6 +56,10 @@ tox = "==3.15.1"
 vulture = "==2.1"
 vyper = "==0.1.0b12"
 isort = "==5.5.2"
+pytest-custom-exit-code = "==0.3.0"
```

Comment:
to suppress specific exit codes. E.g. if pytest doesn't find any test, it exists with exit code 5 instead of 0. This is useful below as we forward the same filters of the main tests to the plugin test, which might not make sense for the latter.